src/libs/antares/study/parts/thermal/cluster.cpp

```diff
@@ -182,6 +186,12 @@ void Data::ThermalCluster::copyFrom(const ThermalCluster& cluster)
     minUpTime = cluster.minUpTime;
     minDownTime = cluster.minDownTime;
 
+    // ramping
+    maxUpwardPowerRampingRate = cluster.maxUpwardPowerRampingRate;
+    maxDownwardPowerRampingRate = cluster.maxDownwardPowerRampingRate;
+    powerDecreaseCost = cluster.powerDecreaseCost;
+    powerIncreaseCost = cluster.powerIncreaseCost;
```

Comment:
```suggestion
    ramping = cluster.ramping;
```

Reply:
Implemented in commit [b9801fe](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1817/commits/b9801feaf09d246c7f326859a24d4c2b97674d71)